addons/l10n_mx/models/account_move.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, _
+from odoo.exceptions import UserError
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    def _get_report_base_filename(self):
```

Comment:
In MX it is illegal print receipts unless you have the implementation of general public tickets (which is not available) @smetl @nim-odoo @jco-odoo 

There is not such thing as **receipt** for a document without invoice/global-invoice in méxico. be careful tickets you close with customer feedback.

This feature is used to use systems for illegal moves.